src/components/SearchBar.js

```diff
@@ -4,54 +4,27 @@ import styled from 'styled-components';
 import { Button, Drawer, Hidden, IconButton } from '@material-ui/core';
 import { Info, Cancel } from '@material-ui/icons';
 
-const SearchBar = () => {
+const SearchBar = props => {
+  const { className } = props;
   const navigate = useNavigate();
   const [searchType, setType] = useState('');
   const [value, setValue] = useState('');
   const [info, openInfo] = useState(false);
   return (
-    <Container>
+    <Container className={className} style={{ position: 'relative' }}>
```

Comment:
this isn't really the styled components way of doing it

for the position relative you can add a parameter if you don't want it in all the Containers (although there's only one so just add that css property to Container)

instead of using className for child selector you can do somehing like this

```
const Kid = styled.div`
    width: 20px;
    height: 20px;
`;
const Container = styled.div`
  ${Kid} {
    display: none;
  }
`
```

Reply:
Ok done. 

https://user-images.githubusercontent.com/24510397/110680045-2594e000-81a6-11eb-876a-987cd372043a.mp4

This is how it looks now. The reset icon is outside the input now because it required re-factoring to use the component twice and it's just simpler like that. I don't think it's a big deal. Overall things look better and mobile header is no longer broken.